aimmo-game/simulation/pickups.py

```diff
@@ -29,8 +29,15 @@ def serialise(self):
 
 class HealthPickup(_Pickup):
     def __init__(self, cell, health_restored=3):
```

Comment:
It's just a default value for the pickup if health_restored isn't specified. It can be changed but it's all down to discussion as to how much goes where tbh.

Reply:
ok.